remappings.txt

```diff
@@ -1,5 +1,7 @@
-@openzeppelin-upgrades/=lib/openzeppelin-contracts-upgradeable/
-@openzeppelin/=lib/openzeppelin-contracts/
+@openzeppelin-upgrades/=lib/eigenlayer-contracts/lib/openzeppelin-contracts-upgradeable/
+@openzeppelin/=lib/eigenlayer-contracts/lib/openzeppelin-contracts/
 ds-test/=lib/ds-test/src/
 eigenlayer-contracts/=lib/eigenlayer-contracts/
 forge-std/=lib/forge-std/src/
+openzeppelin-contracts-upgradeable/=lib/openzeppelin-contracts-upgradeable/
+openzeppelin-contracts/=lib/openzeppelin-contracts/
```

Comment:
It's because we use "@dependency" in the core contracts repo and the default  remapping here.  I did forge remappings > remappings.txt and it added the default one for the oz dependencies which were implicit before

Reply:
ah OK, works for me 👍